definitions/npm/antd_v3.x.x/flow_v0.25.x-v0.103.x/antd_v3.x.x.js

```diff
@@ -601,6 +601,19 @@ declare module "antd" {
 
   declare export class Tooltip extends React$Component<TooltipProps> {}
 
+  declare export class Tree extends React$Component<TreeProps, any> {
+    static TreeNode: typeof React$ComponentClass<AntTreeNodeProps>;
+    static defaultProps: {
+      checkable: boolean;
+      showIcon: boolean;
+      openAnimation: {
```

Comment:
Doesn't seem to fit: https://github.com/ant-design/ant-design/blob/master/components/tree/Tree.tsx#L151-L159